TACC/Core-Portal
# Quick: Correctly hide `"Parameters"` string in app form if none exist

## Overview

This PR correctly does not render the "Parameters" block in the app form if no parameters exist in an app's `parameterSet`.

## Testing

1. Go to https://cep.test/workbench/applications/jupyter-lab-hpc?appVersion=0.0.1
2. Confirm the "Parameters" block does not render an empty block.

## UI
 Before
![Screenshot 2023-07-17 at 5 20 51 PM](https://github.com/TACC/Core-Portal/assets/20326896/749498b2-53aa-41cc-a798-2641ced528c1)


After
![Screenshot 2023-07-17 at 5 21 03 PM](https://github.com/TACC/Core-Portal/assets/20326896/3e83c328-14a6-4849-895e-efe2e7869256)
